Common/GL/GLInterface/EGL.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright 2012 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <array>
+#include <cstdlib>
+
+#include "Common/GL/GLInterface/EGL.h"
+#include "Common/Log.h"
+
+// Show the current FPS
```

Comment:
Looks like this comment got lost somewhere, doesn't seem relevant?

-[Unknown]